source_test.yaml

```diff
@@ -413,3 +413,17 @@
   repo_username: 'git'
   strict_validation: True
 
+- name: 'V8'
+  versions_from_repo: True
+  type: 0
+  ignore_patterns: [ '^(?!V8-advisory).*$' ]
+  repo_url: 'https://github.com/google/chromium-policy-vulnfeed.git'
+  detect_cherrypicks: False
```

Comment:
```suggestion
  detect_cherrypicks: True
```

We need this to ensure we properly treat all branches outside of the specified fix commits as vulnerable.